ci/packages/suite-web.yml

```diff
@@ -17,65 +17,60 @@ variables:
 
 .e2e_script_common: &e2e_script_common
     - npx cypress install
-    - export CYPRESS_ASSET_PREFIX=/wallet
+    - export CYPRESS_ASSET_PREFIX=/web
     - export CYPRESS_baseUrl=${DEV_SERVER_URL}/suite-web/${CI_BUILD_REF_NAME}
     - export TRACK_SUITE_URL=https://track-suite.herokuapp.com
 
-suite-web build:
+suite-web build dev:
     stage: build
     script:
-        - assetPrefix=/suite-web/${CI_BUILD_REF_NAME}/wallet yarn workspace @trezor/suite-web build
+        - assetPrefix=/suite-web/${CI_BUILD_REF_NAME}/web yarn workspace @trezor/suite-web build
     artifacts:
         expire_in: 7 days
         paths:
             - packages/suite-web/build
 
-suite-web build-staging:
+suite-web build beta:
     stage: build
     only:
         <<: *auto_run_branches
     script:
-        - assetPrefix=/wallet yarn workspace @trezor/suite-web build
+        - assetPrefix=/wallet/web yarn workspace @trezor/suite-web build
     artifacts:
         expire_in: 7 days
         paths:
             - packages/suite-web/scripts/s3sync.sh
             - packages/suite-web/build
 
-suite-web build-staging manual:
+suite-web build stable:
     stage: build
-    when: manual
-    except:
+    only:
         <<: *auto_run_branches
     script:
-        - assetPrefix=/wallet yarn workspace @trezor/suite-web build
+        - assetPrefix=/web yarn workspace @trezor/suite-web build
     artifacts:
         expire_in: 7 days
         paths:
             - packages/suite-web/scripts/s3sync.sh
             - packages/suite-web/build
 
 suite-web deploy dev:
-    stage: deploy to dev servers
-    variables:
-        DEPLOY_DIRECTORY: ${DEPLOY_BASE_DIR}/suite-web/${CI_BUILD_REF_NAME}/wallet
-    dependencies:
-        - suite-web build
-        - landing-page build
-    environment:
-        name: ${CI_BUILD_REF_NAME}
-        url: $BASE_REVIEW_URL/${CI_BUILD_REF_NAME}
-    before_script: []
-    script:
-        - cd packages/suite-web
-        - echo "Deploy to dev servers"
-        - mkdir -p ${DEPLOY_BASE_DIR}/suite-web
-        - mkdir -p ${DEPLOY_DIRECTORY} # create build folder
-        - rsync --delete -va build/ "${DEPLOY_DIRECTORY}/"
-        - cd ../landing-page
-        - rsync -abviuzP -va build/ "${DEPLOY_BASE_DIR}/suite-web/${CI_BUILD_REF_NAME}/"
-    tags:
-        - deploy
+  stage: deploy to dev servers
+  variables:
+    DEPLOY_DIRECTORY: ${DEPLOY_BASE_DIR}/suite-web/${CI_BUILD_REF_NAME}/web
+  dependencies:
+    - install and build
+    - suite-web build dev
+  environment:
+    name: ${CI_BUILD_REF_NAME}
+    url: $BASE_REVIEW_URL/${CI_BUILD_REF_NAME}
+  before_script: []
+  script:
+    - mkdir -p ${DEPLOY_DIRECTORY}
+    - cd packages/suite-web
+    - rsync --delete -va build/ "${DEPLOY_DIRECTORY}/"
```

Comment:
i think this should be preserved, otherwise new landing page is not available (i'm seeing old one)
```
- cd ../landing-page-web	
- rsync -abviuzP -va build/ "${DEPLOY_BASE_DIR}/suite-web/${CI_BUILD_REF_NAME}/"
```